app/pages/gsoc/gsoc2022/[id].js

```diff
@@ -1,11 +1,15 @@
 import Head from "next/head";
+import Image from "next/image";
 import { fetchAPI } from "../../../lib/api";
```

Comment:
Fetch of data should be part of the component, should not need to be imported here.

Reply:
Sir, this fetchAPI is needed to fetch the top nav-bar and to make the dynamic routes possible. It is for the strapi APIs.